### PR TITLE
fix: scores of affected did

### PR DESCRIPTION
**Pull request by ruibinx**
{Summary}

Fixes #{issue}

**Type of change**

* [ ] Bug Fix (non-breaking change which fixes an issue)
* [ ] New Feature (non-breaking change which adds functionality)
* [ ] Breaking Change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update
* [ ] This change requires a storage migration
* [ ] This change requires a benchmark

**Checklist**

* [ ] My code follows the style guidelines of this project
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have made corresponding changes to the documentation
* [ ] My changes generate no new warnings
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] New and existing unit tests pass locally with my changes
* [ ] New and existing benchmarks pass locally with my changes
* [ ] Any dependent changes have been merged and published in downstream modules
* [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: pallets/tag/src/migrations.rs [hunk 1]
```diff
@@ -237,1107 +237,74 @@ pub mod v5 {
...
+                let did_id = T::DecentralizedId::from(did.to_fixed_bytes());
+                for tag in tags.clone() {
+                    let tag = tag.as_bytes().to_vec();
+                    // For score read as I32 is zero, there are two situations:
+                    // 1. score is indeed zero.
+                    // 2. score is I8 and can't be read as I32.
+                    // We read those scores as I8 and convert them to I32, so:
+                    // if score is indeed zero, it will not be changed,
+                    // or score is I8, it will be fixed.
+                    let new_personas = crate::PersonasOf::<T>::get(did_id, tag.clone());
```
Comment (coderfengyun) [inline pallets/tag/src/migrations.rs:285]: 这里返回的应当是一个Option，需要处理一下Empty?
另外，undecodable时，这里应该是Empty ?

File: pallets/tag/src/migrations.rs [hunk 2]
```diff
@@ -237,1107 +237,74 @@ pub mod v5 {
...
+                    if new_personas.score() == 0 {
```
Comment (coderfengyun) [inline pallets/tag/src/migrations.rs:286]: 如果undecodable时，new_personas是empty，那么这里的判断也需要处理

File: pallets/tag/src/migrations.rs (hunk 1 above)
Comment (ruibinx) [reply pallets/tag/src/migrations.rs:285]: 这里不是None，设置的是ValueQuery，返回的是Default值

File: pallets/tag/src/migrations.rs (hunk 2 above)
Comment (ruibinx) [reply pallets/tag/src/migrations.rs:286]: 同上

**Review by coderfengyun — APPROVED** [review]

**Pull request merged**